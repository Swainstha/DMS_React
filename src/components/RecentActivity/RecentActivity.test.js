import testSnapFunction from '../../util/testSnapFunction';
import RecentActivity from './index'
import React from 'react';


testSnapFunction("<RecentActivity />", "Snapshot test for RecentActivity ",<RecentActivity /> )