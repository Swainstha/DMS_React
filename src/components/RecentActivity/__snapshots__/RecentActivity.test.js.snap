// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RecentActivity /> Snapshot test for RecentActivity  1`] = `ShallowWrapper {}`;
